src/csv.cpp

```diff
@@ -602,3 +603,38 @@ void write_summary_csv(FILE *csv_file, string square_name, string test,
                          str_num_sites, to_string(energy_capacity)};
   write_to_csv_file(csv_file, line);
 }
+
+std::vector<PitCharacteristics> read_pit_polygons(std::string filename){ 
+  vector<PitCharacteristics> pits;
+  ifstream inputFile(filename);
+  string s;
+  bool header = true;
+  
+  while (getline(inputFile, s)) {
+    if (header) {
+      header = false;
+      vector<string> line = read_from_csv_file(s);
+      continue;
+    }
+    vector<string> line = read_from_csv_file(s);
+    GeographicCoordinate gc =
+        GeographicCoordinate_init(stod(line[1]), stod(line[2]));
+    GeographicCoordinate origin = get_origin(
+        GridSquare_init(convert_to_int(FLOOR(gc.lat)), convert_to_int(FLOOR(gc.lon))),
```

Comment:
This function is just for reading the shape_bound for pits in the constructor module. Wouldn't quite work if part of the standard reservoir csv reading since we read most info from the "pairs" csv in constructor, and this doesn't contain any shape_bound info. Not sure that we want to include shape_bound info in the pairs csv either since we don't need it for existing reservoirs/single pits (that info is in the Shapefiles instead) or the greenfield (since we model those shape_bounds as part of constructor).